packages/flame/lib/src/collisions/broadphase/broadphase.dart

```diff
@@ -5,17 +5,33 @@ import 'package:meta/meta.dart';
 /// by doing a rough estimation of which hitboxes that can collide before their
 /// actual intersections are calculated.
 ///
-/// Currently there is only one implementation of [Broadphase] and that is
-/// [Sweep].
+/// Currently there are two implementations of [Broadphase]:
+/// - [Sweep] - simplest but slowest system, yet nice for small count of objects
+/// - [QuadTree] - work faster but requires additional setup and works only
+///   with fixed-size maps. See [HasQuadTreeCollisionDetection] for details
```

Comment:
```suggestion
/// Currently there are two implementations of [Broadphase]:
/// - [Sweep] is the simplest but slowest system, yet nice for small amounts of hitboxes.
/// - [QuadTree] usually works faster, but requires additional setup and works only
///   with fixed-size maps. See [HasQuadTreeCollisionDetection] for details.
```
Added `usually`, because when I tried it on a map where a lot of tree recalculations had to be done it actually turned out to be slower. But maybe you've done some more recent benchmarks?

Reply:
What kind of tree recalculations? 
Adding and removing components is not cheap and QuadTree creates additional listeners to track updates, whereas Sweep just adds item into list. So it is known... "sacrifice" of performance on component's adding stage with hope to have faster calculations after.

Reply:
A lot of fast moving hitboxes is what I'm talking about, not adding and removing hitboxes.